index.html

```diff
@@ -4564,35 +4572,43 @@ <h3>Verification</h3>
 
         <ol class="algorithm">
           <li>
-Set <var>result</var> to an empty [=map=].
-          </li>
-          <li>
 Ensure that the securing mechanism has properly protected the
 <a>conforming document</a> by performing the following steps:
             <ol class="algorithm">
-              <li>
-Set <var>securingMechanismVerified</var> to the result of executing the
-<a href="#verify-securing-mechanism">securing mechanism verification
-algorithm</a> with <var>inputBytes</var> and <var>mediaType</var> passed as
-inputs to the algorithm.
+             <li>
+Set the |verifyProof| function by using the |inputMediaType| and the
+<a data-cite="?VC-SPECS#securing-mechanisms">Securing Mechanisms</a> section of
+the [[[?VC-SPECS]]] [[?VC-SPECS]], or other mechanisms known to the
+implementation, to determine the cryptographic suite to use when verifying
+the securing mechanism. The |verifyProof| function MUST implement the interface
+described in <a href="#securing-mechanisms"></a>.
```

Comment:
I do not think that is the case. The charter refers to "normative specification of APIs" as being out of scope. At least in my mind (and I believe that was the concept used for the charter) this means defining, e.g., a WebIDL interface, that can be converted, with property names, arguments, etc, into a Javascript or Python interface. At least in other Working Groups at W3C this is what the term "API" is used for. (Typically, those APIs are supposed to be implemented by browsers. Actually, and in my experience with other groups, the very reason these types of clauses are added to charters is exactly to avoid the impression that the underlying technology is an implementation requirement for browsers…)

In this text, if you follow the link, it gets you to [§4.9 Securing Mechanisms](https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1394.html#securing-mechanisms) which remains still fairly high level:

> Securing mechanism specifications MUST provide a verification mechanism that returns only the information in the [conforming document](https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1394.html#dfn-conforming-document) that has been secured. […] A securing mechanism's verification algorithm MUST provide an interface that receives an sequence of bytes ([byte sequence](https://infra.spec.whatwg.org/#byte-sequence) inputBytes) and a media type ([string](https://infra.spec.whatwg.org/#string) inputMediaType) as inputs and returns a verification result with at least the following [items](https://infra.spec.whatwg.org/#struct-item):

How exactly the securing mechanism is built up for a specific case and specific implementation remains open and very high level. Which, I believe, is perfectly fine charter-wise.

For the avoidance of any doubt, maybe the text can formulate the requirement a bit differently, and not use the term "interface".